elasticgraph-schema_definition/lib/elastic_graph/schema_definition/mixins/has_readable_to_s_and_inspect.rb

```diff
@@ -29,6 +29,7 @@ class HasReadableToSAndInspect < ::Module
         def initialize
           if block_given?
             define_method :to_s do
+              # @type self: HasReadableToSAndInspect
```

Comment:
Changed the type of `define_method` where the block requires `@type self: T` annotation.
https://github.com/ruby/rbs/pull/2325